package.json

```diff
@@ -10,6 +10,7 @@
   },
   "dependencies": {
     "axios": "^1.3.5",
+    "dotenv": "^16.0.3",
```

Comment:
we don't need dotenv for this project

Reply:
I will take it out I forgot I installed it when I had the .env issue